src/api/chrome.js

```diff
@@ -215,3 +218,32 @@ const getBadgeCounter = async (initialValue = 0) => {
   }
   return badgeCounter
 }
+
+/**
+ * Retrieves current tab url and validates it. Returns valid tab url otherwise undefined.
+ * @returns the tab url or undefined
+ */
```

Comment:
redundant comment. when the function name already conveys what it does, no need for comments